sn_networking/src/event.rs

```diff
@@ -759,25 +759,10 @@ impl SwarmDriver {
                 if is_new_peer {
                     self.log_kbuckets(&peer);
                     self.send_event(NetworkEvent::PeerAdded(peer));
+                    // todo: do we need this log here? O(n) to calculate .count();
```

Comment:
I think this should be fine in a larger network, this should not be so frequent.